### PR TITLE
Archetypes guidance

### PR DESCRIPTION
**Pull request by maelle**
Update guidance based on new Archetypes structure: the templates home changed; I removed two functions from roblog hence made roblog less prominent (it only has the function for creating the author file; and the post checking functions). 

The current guidance is now outdated.

### CODE REVIEW

File: authortechnical.Rmd [hunk 1]
```diff
@@ -34,19 +31,44 @@ _In Hugo speak, we'd say your post is a [leaf bundle](https://gohugo.io/content-
 
 ## Start the post from a template {#templates}
 
-Prepare your post in your favorite editor in R Markdown **or** Markdown by starting with templates we provide in this book or [from the roblog package](#roblogtemplates).
+Prepare your post in your favorite editor in R Markdown **or** Markdown by starting with templates we provide in this book or using blogdown's handy New Post RStudio addin (recommended approach).
```
Comment (maelle) [inline authortechnical.Rmd]: I'd be strongly in favour of making this the recommended approach because as @steffilazerte noted, handling paths is a bit confusing. With the blogdown addin, there's more setup (installing two packages, installing Hugo) but then less mistakes possible.

**Comment by maelle** [comment]
Note that the blogdown Addin also helps with adding existing tags, which only comes later in the guidance, but I think it's not too bad.

File: authortechnical.Rmd [hunk 2]
```diff
@@ -19,44 +19,65 @@ Briefly, the process is:
...
+knitr::include_graphics("images/blogdownaddin.png", dpi = 25)
+```
 
-[R Markdown template](#templatermd) is to be saved as `/content/blog/YYYY-MM-DD-slug/index.Rmd`.
+  * Enter a title, no need to worry about title case at this stage.
+  * Enter your name if `whoami` wasn't able to guess it.
+  * Choose the correct date.
+  * Enter a new slug if the default one is too long.
+  * Choose "blog" or "technotes" as a Subdirectory from the drop-down menu.
+  * Choose an Archetype, Rmd or md.
```
Comment (stefaniebutland) [inline authortechnical.Rmd]:
```suggestion
  * Choose an Archetype, Rmd or md, from the drop-down menu.
```

File: authortechnical.Rmd [hunk 3]
```diff
@@ -296,4 +318,5 @@ knitr::include_graphics("images/hugochecks.png")
 knitr::include_graphics("images/hugocheckspassed.png")
 ```
 
-
+[^1]: In Hugo speak, we'd say your post is a [leaf bundle](https://gohugo.io/content-management/page-bundles/#leaf-bundles).
+[^2]: If you don't use RStudio you can still use the addin, but the new post will be opened in [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html).
```
Comment (stefaniebutland) [inline authortechnical.Rmd]: I don't quite understand this "the new post will be opened in [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html)"

Does the user need to set their preferred editor that way?

If yes, maybe rephrase

File: authortechnical.Rmd [hunk 4]
```diff
@@ -296,4 +318,5 @@ knitr::include_graphics("images/hugochecks.png")
 knitr::include_graphics("images/hugocheckspassed.png")
 ```
 
-
+[^1]: In Hugo speak, we'd say your post is a [leaf bundle](https://gohugo.io/content-management/page-bundles/#leaf-bundles).
+[^2]: If you don't use RStudio you can still use the addin, but the new post will be opened in the editor returned by [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html), that you might have configured or not.
```
Comment (stefaniebutland) [inline authortechnical.Rmd]:
```suggestion
[^2]: If you don't use RStudio you can still use the addin, but the new post will be opened in the editor returned by [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html), that you might need to configure.
```

**Comment by sckott** [comment]
tried `blogdown:::new_post_addin()` from within R in a terminal, opened up a shiny web app in my browser, then opened the file in my text editor (sublime text) - worked well. I can see maybe using it myself, though I'd be more likely to use it I think if it gave console prompts in the terminal instead of opening a shiny app - Overall though this is great as i imagine 95% of blog post contribs use rstudio anyway

**Comment by maelle** [comment]
Thanks @sckott!
